### ISSUE TITLE
How do we file bugs against the Developer Community website itself?

### ISSUE THREAD

**Issue by MisinformedDNA** [MicrosoftDocs/visualstudio-docs#1990]
I've had some issues with the Developer Community website itself. How do I file bugs against that? Also, how can I make feature suggestions?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 7254052d-2d37-509e-e6c8-d5674ea775ca
* Version Independent ID: a2acc423-07d3-409c-7de1-21dd88e3cdcd
* Content: [Overview: Report a Problem in Visual Studio - Visual Studio](https://docs.microsoft.com/en-us/visualstudio/ide/report-a-problem?view=vs-2017)
* Content Source: [docs/ide/report-a-problem.md](https://github.com/MicrosoftDocs/visualstudio-docs/blob/master/docs/ide/report-a-problem.md)
* Product: **visual-studio-dev15**
* GitHub Login: @seaniyer
* Microsoft Alias: **seiyer**

**Comment by gewarren** [comment]
@seaniyer Can you respond to the question about how to file issues against Developer Community itself? @MisinformedDNA Are you asking about making feature suggestions for Developer Community, or for Visual Studio?

**Comment by seaniyer** [comment]
Hi 

@MisinformedDNA, you can report problems or make suggestions on the Developer Community web site itself at Developer Community.  Please submit your feedback and we'll follow up.

**Comment by weitzhandler** [comment]
What would be the appropriate category?
Any tags I should add to my issue?

![image](https://user-images.githubusercontent.com/2716316/50499619-bf539500-0a53-11e9-98a2-6345507a4c84.png)

**Comment by gewarren** [comment]
@seaniyer Can you provide the requested information here about category and tags?

**Comment by MisinformedDNA** [comment]
Any updates here?

**Comment by neoDD69** [comment]
@MisinformedDNA I think that the site need a huge rethinking for become usable.
Seems to be projects for one scope and bad-modified during the way.
One for all: you cannot see YOUR posts !!!!

embarrassing, especially since it is oriented to professional programmers and not to end users and then it's Microsoft, not you and me!

**Comment by seaniyer** [comment]
@weitzhandler You can submit a new suggestion for Developer Community under Visual Studio.  We don't have a separate category in the dropdown at the moment. Tagging is optional and may not be necessary in this case.

Thanks
Sean

**Comment by seaniyer** [comment]
@neoDD69 

It is true there is no 'Your Posts' view at the moment.  There is an 'Items I Follow' dropdown menu option under your name at the top right of the web site.  'Items I follow' includes your posts and others posts you follow.  Please give it a try if you haven't used it and let us know how it meets / does not meet your needs.  

Thanks!
Sean
  
![image](https://user-images.githubusercontent.com/41308619/51202619-0515b780-18b4-11e9-9444-5a02ca83cff9.png)

**Comment by JohnathanBaird** [comment]
Do I really have to "Report a problem" through Visual Studio to just get help on something I'm working on, especially if it's something other community members could help me with? Why are the only categories for new topics on the forum problems and feature suggestions? What if you just don't know how to do something and need help? That doesn't fall into either provided category, and doesn't make sense why I have to go through VS to actually submit my discussion...

**Comment by gewarren** [comment]
@JohnathanBaird For the type of inquiry you've described, Stack Overflow is probably your best bet.

**Comment by neoDD69** [comment]
@seaniyer, @gewarren
I' tried, but with a poor result:
"Problems (4261) Features (3) Activities " 
Note: :) I never subscribed, or reported, 4261 problems!!! I'm there from 2016 and I suppose this is one of several bugs that the Web site have.

The problem, from our POV (developers), that is very difficult to interact with issues.
- Is hard to follow My Issue
- is hard to follow Issue that I've contribuited
- is hard to know if there are issue on which I can give a hand … or neurons ;)

Seems that is developed on your POV (MS) and not for create an interactive community.

Just a last example:
when I post an ISSUE, to known if anyone have replied to I need to enter in the web site and search my issue with the poor search engine.
No email alert for this normal condition. 
if I want to be e-mailed, I have to suffer hundreds of emails (I tried to activate the selectors but months ago it didn't seem to work well)

**Comment by weitzhandler** [comment]
@neoDD69 I don't complete agree.
When you 'Follow' feedback items, you do get an email for any response on the thread.
I do agree that things could be more accessible and user friendlier.

**Comment by seaniyer** [comment]
@neoDD69

Hi Davide, 
Thanks for your input that it is hard to follow your own issues.  We are looking into improving the experience.  I notice that you are following three large topics C#, editor and Windows forms.  Following topics results in following the feedback within the topics.  You can unfollow those topics to prevent further increase in the items followed. 

I also saw you may have opened another account.  We can take action to clean up the old account to reset the following count down to just what you contributed, but that may not be necessary since you have a new account.  Let me know if you'd like us to try to do that.

Thanks
Sean
Visual Studio Feedback Team

**Comment by nschonni** [comment]
@seaniyer is there actually a support channel though? I also am supposed to get daily digests, but I just get a dozen emails every Friday instead. Because of the usability problems, I don't bother using DevCom since it just seems like Connect all over again.

**Comment by neoDD69** [comment]
@seaniyer,
It's a good news that someone look at the usability and productivity of the web site. It's years that I try to evidence some lacks.

Your are right some months ago I tried to un follow the "groups" but without any result ... all "followed" issue remain followed!!
Can you UNSCRIBE my account from any issue? For me, user, there is not a way to do this (another bug)

About my "second" account ... the "problem" was/is Windows 10 ;) 
I never logon or manually added the second account to dev website ... I feel that is a issue linked to "federation authentication".
My primary account to interact with MS is @ outlook.it but my company user is @ spinwave.it... one day I found they "mixed".
Yes, please "We can take action to clean up the old account to reset the following count down to just what you contributed" on the old account (on outlook.it).
And you can remove the other account @ spinwave created by the site it-self.

I agree with what @nschonni wrote:
1 - a channel/group to evidence issues and suggestion for the web site
2 - someone to debug and finish the website (find my hard comments about the site in past months)

Thanks for your attention.
Davide.

> @neoDD69
> Hi Davide,
> Thanks for your input that it is hard to follow your own issues.  We are looking into improving the experience.  I notice that you are following three large topics C#, editor and Windows forms.  Following topics results in following the feedback within the topics.  You can unfollow those topics to prevent further increase in the items followed.
> I also saw you may have opened another account.  We can take action to clean up the old account to reset the following count down to just what you contributed, but that may not be necessary since you have a new account.  Let me know if you'd like us to try to do that.
> Thanks
> Sean
> Visual Studio Feedback Team

**Comment by seaniyer** [comment]
> @seaniyer is there actually a support channel though? I also am supposed to get daily digests, but I just get a dozen emails every Friday instead. Because of the usability problems, I don't bother using DevCom since it just seems like Connect all over again.

@nschonni - The best channel to reach us at this time is to submit a feedback via Visual Studio Feedback tool or Developer Community site. Regarding frequency of emails, you can see manage them under your profile and preferences page on DevCom (screenshot showing navigation to the page below). Let me know if you have any trouble finding it.
![1](https://user-images.githubusercontent.com/41308619/62328749-c4a11400-b468-11e9-8147-ed2d20a95eb9.jpg)

I'm sorry to hear that you've run into usability issues.  I'd like to find out more. Can we setup a time for a chat and get some details on your experience?

Sean
Visual Studio Feedback Team

**Comment by grockland** [comment]
There is no doubt the MS Developer Community website for submission of suggestions or bugs works abominably. Apparently there is no motivation to improve it.  I guess we have to mark it as Closed.
